src/app/cube/home/learning-object-info/learning-outcomes/learning-outcomes.component.scss

```diff
@@ -1,15 +1,77 @@
-.content{
+@import 'vars.scss';
+
+.content {
     margin-top: 50px;
     margin-right: 50px;
+    padding-left: 0px;
     display: flex;
     flex-direction: column;
-    .taxonomy{
-      margin-bottom: 400px;
+    
+    .blooms-cards {
+        display: flex;
+        flex-direction: row;
+        justify-content: space-evenly;
+        gap: 25px;
+        color: $light-blue;
+        padding-bottom: 300px;
+
```

Comment:
Adding a 50px bottom margin beneath the taxonomy class will better match the spacing in the other sections.

Reply:
The bottom padding is now 350px. I'm not sure if that's what you wanted with this.